MovingBlocks/Terasology
# JoshariasSurvival is now selected as the first default gameplay template

### Contains

Provides a solution to issue #2352, and also adds an "Enable All" button to the module selection screen, which selects all available mods.

**EDIT**: Reverted "Enable All" changes.

### How to test

- Delete the user config file (somehow, perhaps by cloning a fresh copy of the engine?)
- Make sure "JoshariasSurvival" template is downloaded and included.
- Load up the game
- Go to the "Create Game" screen.
- It should be selected by default.

### Credits

@Cervator: For giving me advice on this matter.
